src/utils.js

```diff
@@ -550,8 +550,8 @@ export function calculateVesting(vestingInfo, amount, genesisDate) {
     .fill(0)
     .map((_, index) => {
       const date = new Date(genesisDate)
```

Comment:
`new Date` also uses local time, not UTC

Reply:
I agree that using `Date` here offers no advantage. We can simply go with:
```js
const date = genesisDate + delay + cliff + installmentLength * index
```